packs/data/pfs-season-1-bestiary.db/quagmire.json

```diff
@@ -68,7 +68,7 @@
                 "value": 1
             },
             "reset": "<p>The hazard still submerges anyone who walks in, but the surface doesn't become hidden again until it settles over the course of 24 hours</p>",
-            "routine": "<p><span class=\"pf2-icon\">1</span> On its initiative, the quagmire pulls each creature within it 1 square toward the center of the quagmire and 1 step further submerged; a creature that is submerged up to its waist becomes submerged up to its neck, and a creature that is submerged up to its neck is pulled under and has to hold its breath to avoid suffocation.</p>\n<p>A creature in the quagmire can attempt a @Check[type:athletics|dc:17|name:Swim Out of Quagmire|traits:environmental,hazard,action:swim] check to Swim to either raise itself by 1 step if its submerged to its neck or worse, or to swim out of the quagmire if its submerged up to only its waist. On a critical failure, the creature is pulled down 1 step. A creature that Swims out of the quagmire escapes the hazard and is @UUID[Compendium.pf2e.conditionitems.Prone]{Prone} adjacent to the quagmire. Other creatures can @UUID[Compendium.pf2e.actionspf2e.Aid]{Aid} the creature, typically by using a rope or similar tool, or attempt to pull a creature within their reach out with a @Check[type:athletics|dc:17|name:Pull Out of Quagmire|traits:environmental,hazard,action:aid] check, with the same results as if the creature attempted the check.</p>"
+            "routine": "<p>(1 action) On its initiative, the quagmire pulls each creature within it 1 square toward the center of the quagmire and 1 step further submerged; a creature that is submerged up to its waist becomes submerged up to its neck, and a creature that is submerged up to its neck is pulled under and has to hold its breath to avoid suffocation.</p>\n<p>A creature in the quagmire can attempt a @Check[type:athletics|dc:17|name:Swim Out of Quagmire|traits:environmental,hazard,action:swim] check to Swim to either raise itself by 1 step if its submerged to its neck or worse, or to swim out of the quagmire if its submerged up to only its waist. On a critical failure, the creature is pulled down 1 step. A creature that Swims out of the quagmire escapes the hazard and is @UUID[Compendium.pf2e.conditionitems.Prone]{Prone} adjacent to the quagmire. Other creatures can @UUID[Compendium.pf2e.actionspf2e.Aid]{Aid} the creature, typically by using a rope or similar tool, or attempt to pull a creature within their reach out with a @Check[type:athletics|dc:17|name:Pull Out of Quagmire|traits:environmental,hazard,action:aid] check, with the same results as if the creature attempted the check.</p>"
```

Comment:
```suggestion
            "routine": "<p>(1 action) On its initiative, the quagmire pulls each creature within it 1 square toward the center of the quagmire and 1 step further submerged; a creature that is submerged up to its waist becomes submerged up to its neck, and a creature that is submerged up to its neck is pulled under and has to hold its breath to avoid suffocation.</p>\n<p>A creature in the quagmire can attempt a @Check[type:athletics|dc:17|name:Swim Out of Quagmire|traits:environmental,hazard,action:swim] check to Swim to either raise itself by 1 step if its submerged to its neck or worse, or to swim out of the quagmire if its submerged up to only its waist. On a critical failure, the creature is pulled down 1 step. A creature that Swims out of the quagmire escapes the hazard and is @UUID[Compendium.pf2e.conditionitems.Prone]{Prone} adjacent to the quagmire. Other creatures can @UUID[Compendium.pf2e.actionspf2e.Aid]{Aid} the creature, typically by using a rope or similar tool, or attempt to pull a creature within their reach out with a @Check[type:athletics|dc:17|name:Pull Out of Quagmire] check, with the same results as if the creature attempted the check.</p>"
```